src/objects.rs

```diff
@@ -340,6 +345,32 @@ impl Id<StopArea> for StopPoint {
 }
 impl_codes!(StopPoint);
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct FeedInfo {
+    #[serde(rename = "feed_info_param")] pub info_param: String,
+    #[serde(rename = "feed_info_value")] pub info_value: String,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Calendar {
+    #[serde(rename = "calendar_id")] pub id: String,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")] pub monday: bool,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")] pub tuesday: bool,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")] pub wednesday: bool,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")] pub thursday: bool,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")] pub friday: bool,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")] pub saturday: bool,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")] pub sunday: bool,
+    pub start_date: String,
+    pub end_date: String,
```

Comment:
We'll need to type that.

Reply:
this can be done in another PR?

Reply:
I wrote an issue https://github.com/CanalTP/navitia_model/issues/17